packages/uswds-core/src/styles/mixins/typography/usa-content-styles.scss

```diff
@@ -3,17 +3,16 @@
 @use "./usa-prose" as *;
 @use "./usa-list" as *;
 @use "./usa-table-styles" as *;
-@use "../../placeholders/typography" as *;
 
 @mixin usa-paragraph-style {
   p {
-    @extend %usa-prose-p;
+    @include typeset-p;
```

Comment:
The usa-prose mixins are all redundant with the typeset mixins, so I replaced them here.